apps/cowswap-frontend/src/modules/limitOrders/hooks/useHandleOrderPlacement.ts

```diff
@@ -85,13 +85,13 @@ export function useHandleOrderPlacement(
         // Reset override after successful order placement
         setPartiallyFillableOverride(undefined)
       })
-      .catch((error: Error) => {
+      .catch((error) => {
         if (error instanceof PriceImpactDeclineError) return
 
         if (error instanceof OperatorError) {
           tradeConfirmActions.onError(error.message)
         } else {
-          tradeConfirmActions.onDismiss()
+          tradeConfirmActions.onError('Unknown error. Please try again.')
```

Comment:
We don't want to try to show the `error.message` if present here?

Reply:
Not sure we do. If it's not a known error type, the actual message contents might be too technical for most people.
Anyway, before we weren't showing anything, so this is a generic error msg instead of nothing.

Reply:
Done